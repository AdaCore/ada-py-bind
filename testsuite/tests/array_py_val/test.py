import gen

p = gen.Plot()

p.points[1].coords[1] = 12
p.points[1].coords[2] = 11
p.points[1].coords[3] = 10

print p.points[1].coords[1]
print p.points[1].coords[2]
print p.points[1].coords[3]
