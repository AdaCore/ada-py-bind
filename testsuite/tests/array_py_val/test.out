12
11
10
